src/app/fddev/dev.c

```diff
@@ -67,6 +67,10 @@ update_config_for_dev( config_t * const config ) {
      validator will get stuck forever. */
   config->consensus.wait_for_vote_to_start_leader = 0;
 
+  /* We have to wait until we get a snapshot before we can join a second
```

Comment:
How does Solana Labs do this?

Reply:
As far as I know, this is still required when using stock Solana Labs for everything.